firmware-management/updated/CMakeLists.txt

```diff
@@ -4,19 +4,21 @@ cmake_minimum_required(VERSION 3.5)
 
 option(RUN_CODE_CHECKS OFF)
 
-if(RUN_CODE_CHECKS)
-    set(CMAKE_CXX_CLANG_TIDY clang-tidy;-checks=*,-clang-analyzer-cplusplus-*,clang-analyzer-*)
-endif(RUN_CODE_CHECKS)
-
-add_executable(updated ${CMAKE_CURRENT_SOURCE_DIR}/updated.cpp ${CMAKE_CURRENT_SOURCE_DIR}/init.cpp)
-target_include_directories(updated PRIVATE ${CMAKE_CURRENT_BINARY_DIR})
+# INTERFACE library for compile options that are not for enabling warnings.
```

Comment:
I think we should add the copyright notice and SPDX License identifier to this file.

Reply:
4c1a07c